src/test/java/com/pwn9/filter/MockPlugin.java

```diff
@@ -163,4 +193,8 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
     public List<String> onTabComplete(CommandSender sender, Command command, String alias, String[] args) {
         return null;
     }
+
+    void addPlayer(MockPlayer player){
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/AddstarMC/PwnFilter/pullRequest?prid=6040651)